### DISCUSSION TITLE
I am not able to use my Azure benefits.

### DISCUSSION THREAD

**Discussion by YashG2009** [community/community#144427] [GitHub Education]
### Hi everyone,

Azure says that I am not eligible, though I was verified a time ago. I have an official university account and still they are doing this. I don't know what to do so, just please guide me. Yash@2009221. 
Unable to confirm your University ID is the message I keep seeing

**Comment by Yaser-123** [comment]
@YashG2009 
It seems like there may be an issue with how your university account is linked or verified. I recommend checking that your university's domain is correctly listed in Azure's eligibility system and ensuring that your account information matches what's required for verification. If the issue persists, contacting Azure support or your university's IT department might help resolve it.
